packages/dds/tree/src/test/shared-tree/opSize.bench.ts

```diff
@@ -70,12 +69,11 @@ const childrenFieldKey: FieldKey = brand("children");
 function initializeTestTree(tree: ISharedTree, state: JsonableTree = initialTestJsonTree) {
 	tree.storedSchema.update(fullSchemaData);
 	// inserts a node with the initial AppState as the root of the tree
-	tree.runTransaction((forest, editor) => {
-		const writeCursor = singleTextCursor(state);
-		const field = editor.sequenceField(undefined, rootFieldKeySymbol);
-		field.insert(0, writeCursor);
-		return TransactionResult.Apply;
-	});
+	tree.transaction.start();
+	const writeCursor = singleTextCursor(state);
+	const field = tree.editor.sequenceField(undefined, rootFieldKeySymbol);
+	field.insert(0, writeCursor);
+	tree.transaction.commit();
```

Comment:
this whole edit is a single insert, so it seems unneeded to wrap it in a transaction, right?

Reply:
Agreed, thanks.